src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -80,19 +81,24 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 			} else if ((keyEvent.ctrlKey || keyEvent.metaKey) && keyEvent.keyCode === KeyCode.KEY_B) {
 				// Bold text
 				preventDefaultAndExecCommand(e, 'bold');
+				this.cellModel.notebookModel.sendNotebookTelemetryActionEvent(TelemetryKeys.NbTelemetryAction.WYSIWYGKeyboardAction, { action: 'BOLD' });
```

Comment:
action is already a reserved property when sending action events - https://github.com/Microsoft/azuredatastudio/blob/1c9ba64ee0ad79bbcdfb704038ba4cd7fd8bc986/src/sql/platform/telemetry/common/adsTelemetryService.ts#L133

Reply:
Updated the property name to transformAction, thanks Charles!